docs/developers-guide-drivers.md

```diff
@@ -0,0 +1,39 @@
+# Metabase Driver Development Guide
+
+As of Metabase 0.32, we now support community-built database drivers!
+
+Several folks have already started to build drivers for database integrations that Metabase doesn't natively support. While we don't specifically endorse any of these drivers, we thought it would be helpful to have a central place to list them.
+
+Want to build your own driver? Take a look at the [driver development](#driver-development) section below.
+
+## Community Database Drivers
```

Comment:
If the main title is changed as I suggested, this one should change to be more different from the main title, like "How to use a community-built driver."